docs/e2e.md

```diff
@@ -97,3 +107,8 @@ Pre-requisite for KinD clusters: please add in your local `/etc/hosts` file `127
   ```
   poetry run pytest -v -s ./tests/e2e -m openshift --timeout=1200
   ```
```

Comment:
We can add this cmd `poetry run pytest -v -s ./tests/e2e -m 'nvidia_gpu'` in test phase for kind and openshift clsuter to execute all gpu tests in one run

Reply:
Well, OpenShift tests doesn't need any marker as all tests can be executed.
By default when `nvidia_gpu` marker is not used then the tests are included.

Reply:
Understood